compiler/msgs.nim

```diff
@@ -486,6 +486,31 @@ proc formatMsg*(conf: ConfigRef; info: TLineInfo, msg: TMsgKind, arg: string): s
               else: ErrorTitle
   conf.toFileLineCol(info) & " " & title & getMessageStr(msg, arg)
 
+proc colorError(s: string, color: ForegroundColor): string =
```

Comment:
add a tests/compiler/tmsgs.nim test file with:
```nim
block: # colorError
  add tests here for `colorError`
```
(make some procs public if needed)

=> will make it easier to debug / improve it once we have tests